packages/editor-ui/src/composables/useWorkflowHelpers.ts

```diff
@@ -1004,9 +1004,19 @@ export function useWorkflowHelpers(options: { router: ReturnType<typeof useRoute
 			if (resetWebhookUrls) {
 				workflowDataRequest.nodes = workflowDataRequest.nodes!.map((node) => {
 					if (node.webhookId) {
-						node.webhookId = uuid();
+						const newId = uuid();
+						node.webhookId = newId;
+						node.parameters.path = newId;
 						changedNodes[node.name] = node.webhookId;
 					}
+					// Also update the webhookSuffix if it exists in wait nodes
+					if (
+						node.parameters.options &&
+						typeof node.parameters.options === 'object' &&
+						'webhookSuffix' in node.parameters.options
+					) {
+						node.parameters.options.webhookSuffix = `${node.parameters.options.webhookSuffix}-copy`;
+					}
```

Comment:
Is this necessary? The Wait node Webhook URL path is generated on execution so it shouldn't clash due to duplication

Reply:
true. I did not think of this, and was the one who asked milorad to add this to the tests.
@MiloradFilipovic please ignore what I said about the suffix.